service/src/main/java/crawlercommons/urlfrontier/service/memory/URLQueue.java

```diff
@@ -13,6 +13,8 @@
 
 public class URLQueue extends PriorityQueue<InternalURL> implements QueueInterface {
 
+    private static final long serialVersionUID = 1787185139558575689L;
```

Comment:
we don't specify serials in any of the code - let's not do it here

Reply:
Ok, will remove it